src/main/java/cc/hyperium/mixinsimp/client/GlStateModifier.java

```diff
@@ -59,7 +77,7 @@ public void setTexture(int id) {
         }
 
         if (theArray == null || textureNamefield == null || activeTextureUnit == -1) {
-            System.out.println("Hey staff team if you see this, there was no hope but this message band aids it so hello. ");
+            System.out.println("[GlStateModifer] Problem: no hope");
```

Comment:
Not descriptive enough. Either remove the message entirely or keep the original.